src/test/groovy/com/datadog/ddwaf/NativeWafHandleTest.groovy

```diff
@@ -0,0 +1,44 @@
+package com.datadog.ddwaf
+
+import org.junit.Test
+
+class NativeWafHandleTest extends WafTestBase {
+    NativeWafHandle nativeWafHandle = null
+
+    @Test
+    void 'native waf handle is online'() {
+        builder.addOrUpdateRuleConfig('enya', ARACHNI_ATOM_V1_0, ruleSetInfo)
+        nativeWafHandle = builder.buildNativeWafHandleInstance(null)
+
+        assert nativeWafHandle.online
+        nativeWafHandle.destroy()
+    }
+
+    @Test
+    void 'native waf handle has addresses'() {
```

Comment:
`native waf handle` -> `waf handle`. The fact that we have some class wrapping a C pointer is really not a concern for outside users.